### PR TITLE
[MRG] ENH: plotting: better dimming on white bg

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
A better approach to dimming on white bg: lighten more the background.

Old behavior:
screenshot_dim_old

New behavior:
screenshot_dim_new

### CODE REVIEW

**Comment by bthirion** [comment]
Looks good. One small point is: what is the range of values for the dim argument. I have the impression that this is [-1, 1], but I don't see it in dosctrings.

**Comment by GaelVaroquaux** [comment]
@bthirion : comment addressed. Thanks!

Everybody: can I haz reviews / merge?

**Comment by KamalakerDadi** [comment]
LGTM

File: nilearn/plotting/img_plotting.py [hunk 1]
```diff
@@ -446,7 +446,10 @@ def plot_anat(anat_img=MNI152TEMPLATE, cut_coords=None,
             to matplotlib.pyplot.savefig.
         dim: boolean or float, optional
             Dimming factor applied to background image. If True, automatic
-            heuristics are applied. Accepted float values are between -1 and 1.
+            heuristics are applied. Accepted float values, where at
```
Comment (lesteve) [inline nilearn/plotting/img_plotting.py]: at -> a

File: nilearn/plotting/img_plotting.py [hunk 2]
```diff
@@ -613,7 +616,9 @@ def plot_roi(roi_img, bg_img=MNI152TEMPLATE, cut_coords=None,
             magically by analysis of the image.
         dim: boolean or float, optional
             Dimming factor applied to background image. If True, automatic
-            heuristics are applied. Accepted float values are between -1 and 1.
+            heuristics are applied. Accepted float values, where at
+            typical span is -1 to 1 (-1 = increase contrast; 1 = decrease
+            contrast), but larger values can be used for a more
```
Comment (lesteve) [inline nilearn/plotting/img_plotting.py]: You are missing the "pronounced effect" line.

File: nilearn/plotting/img_plotting.py [hunk 3]
```diff
@@ -613,7 +616,9 @@ def plot_roi(roi_img, bg_img=MNI152TEMPLATE, cut_coords=None,
             magically by analysis of the image.
         dim: boolean or float, optional
             Dimming factor applied to background image. If True, automatic
-            heuristics are applied. Accepted float values are between -1 and 1.
+            heuristics are applied. Accepted float values, where at
```
Comment (lesteve) [inline nilearn/plotting/img_plotting.py]: at -> a

File: nilearn/plotting/img_plotting.py [hunk 4]
```diff
@@ -720,7 +725,9 @@ def plot_prob_atlas(maps_img, anat_img=MNI152TEMPLATE, view_type='auto',
             savefig.
         dim: boolean or float, optional
             Dimming factor applied to background image. If True, automatic
-            heuristics are applied. Accepted float values are between -1 and 1.
+            heuristics are applied. Accepted float values, where at
```
Comment (lesteve) [inline nilearn/plotting/img_plotting.py]: at -> a

File: nilearn/plotting/img_plotting.py [hunk 5]
```diff
@@ -720,7 +725,9 @@ def plot_prob_atlas(maps_img, anat_img=MNI152TEMPLATE, view_type='auto',
...
+            typical span is -1 to 1 (-1 = increase contrast; 1 = decrease
+            contrast), but larger values can be used for a more
```
Comment (lesteve) [inline nilearn/plotting/img_plotting.py]: Missing "pronounced effect"

**Comment by lesteve** [comment]
Merging thanks ! I'll address the minor cosmetic comments in master directly.

**Comment by GaelVaroquaux** [comment]
> Merging thanks ! I'll address the minor cosmetic comments in master directly.

Thanks a lot. Sorry for the delay. I promise that I was planning to
address them.

**Pull request merged**